Tasks/DotNetCoreInstallerV0/dotnetcoreinstaller.ts

```diff
@@ -138,17 +204,48 @@ class DotnetCoreInstaller {
         return cachedDir;
     }
 
+    /**
+     * Search for files from a given path.
+     * @param dir the dir we use to search for files
+     * @param searchPattern the regex search pattern the file name must match. The default matches on all names!
+     * @param recursive search recursive for the files. The default is true!
+     */
+    private getFiles(dir: string, searchPattern?: string, recursive?: boolean): Array<string> {
+        searchPattern = typeof searchPattern !== 'undefined' ? searchPattern : ".*";
+        recursive = typeof recursive !== 'undefined' ? recursive : true;
+
+        const subPaths = fileSystem.readdirSync(dir);
+        let matchedFiles = new Array<string>();
+        for (const subPath of subPaths) {
+            const res = path.resolve(dir, subPath);
+            if (
+                (fileSystem.statSync(res)).isDirectory() &&
+                recursive) {
+                matchedFiles = matchedFiles.concat(this.getFiles(res, searchPattern));
+            } else {
+                if (res.match(searchPattern)) {
+                    matchedFiles.push(res);
+                }
+            }
+        }
+        return matchedFiles;
+    }
+
     private packageType: string;
-    private version: string;
+    private version?: string;
+    private useGlobalJson: boolean = false;
     private cachedToolName: string;
     private arch: string;
+    private workingDirectory: string;
 }
 
 async function run() {
     let packageType = tl.getInput('packageType', true);
-    let version = tl.getInput('version', true).trim();
+    let version = tl.getInput('version', false).trim();
+    let useGlobalJson = tl.getBoolInput('useGlobalJson');
+    let workingDirectory = tl.getPathInput("workingDirectory", false)
```

Comment:
resolve workingDirectory here itself

Reply:
I think I have resolve this. If not reopen this conversion.